packages/babel-parser/test/fixtures/typescript/types/invalid-import-type-trailing-comma/options.json

```diff
@@ -0,0 +1,6 @@
+{
+  "plugins": [
+    "typescript"
+  ],
+  "throws": "Unexpected token, expected \"{\" (1:33)"
```

Comment:
Can we have better recovery here? Especially given that a trailing comma would normally be valid in a dynamic import.

Reply:
We could have way better recovery if we parse as an assignment expression as it would be in JS and deeply validate all the AST. But the intention of this PR is to match the `tsc` behaviour. If `tsc` does not recover from this production then we throw the same error as well.

The type-import is a TS thing so they have the final interpretation of the language syntax. It would be great if it can match the JS, but I am fine if it does not.

That said, if tsc will fix https://github.com/microsoft/TypeScript/issues/61489, we can easily restore the trailing comma support.